.travis.yml

```diff
@@ -1,39 +1,27 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "4"
 sudo: false
 matrix:
   include:
-    - node_js: "0.10"
       env: UNDERSCORE=1.4.4 BACKBONE=1.0
```

Comment:
It looks like it works https://travis-ci.org/marionettejs/backbone.radio/builds/116464309 but I'll update it